catalog/app/containers/Admin/Settings.tsx

```diff
@@ -0,0 +1,257 @@
+import * as FF from 'final-form'
+import * as R from 'ramda'
+import * as React from 'react'
+import * as RF from 'react-final-form'
+import * as M from '@material-ui/core'
+
+import SubmitSpinner from 'containers/Bucket/PackageDialog/SubmitSpinner'
+import * as Notifications from 'containers/Notifications'
+import * as CatalogSettings from 'utils/CatalogSettings'
+import * as validators from 'utils/validators'
+
+import * as Form from './Form'
+
+const useNavLinkEditorStyles = M.makeStyles((t) => ({
+  section: {
+    alignItems: 'center',
+    display: 'flex',
+    marginTop: t.spacing(1),
+  },
+  fields: {
+    display: 'flex',
+    flexDirection: 'column',
+    marginRight: t.spacing(2),
+    overflow: 'hidden',
+  },
+  field: {
+    display: 'flex',
+  },
+  fieldName: {
+    ...t.typography.body2,
+    flexShrink: 0,
+    fontWeight: t.typography.fontWeightMedium,
+    width: 50,
+  },
+  fieldValue: {
+    ...t.typography.body2,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
```

Comment:
If you intended to cut long labels, it doesn't work

Reply:
i forgot `whiteSpace: 'nowrap'`, thx